packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -679,14 +684,14 @@
 
 @mixin bpk-radio__input {
   position: absolute;
-  top: ($bpk-spacing-base - $bpk-spacing-md) / 2;
+  top: 0;
   left: 0;
-  width: $bpk-spacing-md;
-  height: $bpk-spacing-md;
+  width: $bpk-spacing-base - ($bpk-one-pixel-rem * 4);
+  height: $bpk-spacing-base - ($bpk-one-pixel-rem * 4);
```

Comment:
As this is repeated for `width` and `height`, perhaps we should declare a variable for this (not a token, just a variable local to this mixin)

Reply:
Sounds good